Community/Tdarr_Plugin_MC93_Migz1FFMPEG.js

```diff
@@ -335,8 +366,75 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
     response.preset = '-c:v vp8_cuvid';
   }
 
-  response.preset += `${genpts}, -map 0 -c:v hevc_nvenc -cq:v 19 ${bitrateSettings} `
-  + `-spatial_aq:v 1 -rc-lookahead:v 32 -c:a copy -c:s copy -max_muxing_queue_size 9999 ${extraArguments}`;
+  // Determine if GPU is present and which gpu to use based on utilization of each GPU
+  response.infoLog += 'Running nvidia-smi to see if a GPU is available and selecting the GPU with less utilization\n';
+  let gpu_num = -1;
+  let gpu_util = 100000;
+  let result_util = 0;
+  let gpu_count = '';
+  let gpu_res = '';
+  const gpus_to_exclude = inputs.exclude_gpus.toLowerCase().split(',');
+  try {
+    gpu_res = execSync('nvidia-smi --query-gpu=name --format=csv,noheader');
+    gpu_res = gpu_res.toString().trim();
+    gpu_res = gpu_res.split(/\r?\n/);
+    /* When nvidia-smi returns an error it contains 'nvidia-smi' in the error
+      Example: Linux: nvidia-smi: command not found
+               Windows: 'nvidia-smi' is not recognized as an internal or external command, 
+                   operable program or batch file. */
+    if (!gpu_res[0].includes('nvidia-smi')) {
+      gpu_count = gpu_res.length;
+    } else {
+      gpu_count = -1;
+    }
+  } catch (error) {
+    response.infoLog += 'Error in reading nvidia-smi output!\n';
+    response.infoLog += error.message;
+    gpu_count = -1;
```

Comment:
Perhaps initialize gpu_count with `-1` instead of having to set it to -1 in two spots?

Reply:
Done! :)